client/my-sites/plans/jetpack-plans/product-store/wpcom-styles.scss

```diff
@@ -0,0 +1,41 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+// custom properties for Jetpack
+:root {
+	// border-radius
+	--jetpack-corners-sharp: 0;
+	--jetpack-corners-soft: 4px;
+}
+
+.jetpack-product-store {
+
+	/**
+	 * Import Jetpack cloud styles but make them scoped to jetpack-product-store.
+     * Otherwise it will mess up Calypso Blue styles.
+	 */
+	/* stylelint-disable-next-line no-invalid-position-at-import-rule */
+	@import 'calypso/jetpack-cloud/style';
```

Comment:
What a great way to handle this! I don't think I even knew this was possible! Great thinking and good solution! 🙌